declarations.d.ts

```diff
@@ -867,7 +867,7 @@ interface ILiveSyncServiceBase {
 	 * If watch option is not specified executes full sync
 	 * If watch option is specified executes partial sync
 	 */
-	sync(data: ILiveSyncData[], projectId: string, filePaths?: string[]): Promise<void>;
+	sync(data: ILiveSyncData[], projectId: string, filePaths: string[], projectFilesConfig: IProjectFilesConfig): Promise<void>;
```

Comment:
Why is this changed? The `sync`  method must have `filePaths` as optional argument, as they are passed only when we are not in full sync mode